rules.bolt

```diff
@@ -227,7 +227,36 @@ path /inventory {
     read() { true }
 }
 
+// --- New Members Section ---
+
 path /members {
-    read() { isAdmin() }
-    write() { true }
+    read() {
+        isAdmin() || isCurrentUser(data.child('uid').val())
+    }
+    path /{members} is Members {
+        write() { true }
+    }
+}
+
+type Members {
```

Comment:
```suggestion
type Member {
```

Reply:
I didn't do that because there is one `type member` present in line 188 :sweat_smile:

Reply:
Make this ClubMember then

Reply:
Okay :+1: